PVEntityGenerator/PVEntityGenerator.csproj

```diff
@@ -25,7 +25,7 @@
     </FileUpgradeFlags>
     <UpgradeBackupLocation>
     </UpgradeBackupLocation>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
it should not be required to update the dependency here in this PR - if required this should go to a separate PR.